pypeit/spectrographs/bok_bc.py

```diff
@@ -0,0 +1,364 @@
+"""
+Module for WHT/ISIS specific methods.
```

Comment:
Please change the docs to indicate this is Bok and not WHT/ISIS.

Reply:
This I have fixed. Just let me know what to do with the previous one.